ncm-ceph/src/main/pan/components/ceph/config.pan

```diff
@@ -5,4 +5,17 @@
 
 unique template components/${project.artifactId}/config;
 
-include 'components/${project.artifactId}/config-rpm';
+variable CEPH_SCHEMA_VERSION ?= 'v1';
+
+include format('components/${project.artifactId}/%s/schema', CEPH_SCHEMA_VERSION);
+
```

Comment:
can you add site overrid like https://github.com/quattor/maven-tools/pull/170/files#diff-fb9fe79c1696b94dcb0e714774c01f61R71